Migrated Project from .Net 3.1 to .Net 6.0

D1mon007 opened the pull request:
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/EventsExpress/EventsExpress/projects/2)


## Code reviewers

@iuriikhrystiuk 
@sand0 

### Second Level Review

@YourFriendlyNeighbourPasha 

## Summary of issue

Migrate Project from .Net 3.1 to .Net 6.0

## Summary of change

Update the target framework, update package references, and fix the bugs that appears after changes

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

D1mon007 committed:
Migrated Project from .Net 3.1 to .Net 6.0

iuriikhrystiuk commented on EventsExpress.Core/EventsExpress.Core.csproj:
```diff
@@ -1,29 +1,38 @@
     <ProjectReference Include="..\EventsExpress.Db\EventsExpress.Db.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```
We have a separate file for these imports. Remove this import please.

iuriikhrystiuk reviewed (commented):
Do not use preview or beta versions of packages please.

D1mon007 committed:
Fix imports

D1mon007 committed:
Fix

D1mon007 committed:
Fix

D1mon007 committed:
Fixed tests

D1mon007 committed:
Dotnet 6 test (#961)

* Fix. (#909)

Co-authored-by: Hryn Oleksandr <Apostall107@users.noreply.github.com>

* Issue page date picker bug/ Issue page unfinite loading on wrong date enter (#908)

* Changed date formate to prevent bug.

* Date parse for date picker added.

* Naming and code smell improvement.

* Naming fixed.

* Use of moment.

* *now parseEuDate uses  moment.

*Now labels works correctly with fields:
Label is shown as default value in  field and moves up on touch and if value is entered.

Co-authored-by: Hryn Oleksandr <Apostall107@users.noreply.github.com>

* Fixed twitter login. (#911)

* #897 - Error on new event creation attempt, #903 - UI app does not handle 500 internal error correctly (#913)

* Changed response object for 500 Internal Server Error, adapted action-helpers.js for new response object, made an output of error in event-form.js

* Refactored action-helpers.js, changed error handling in edit-event

* Fixed issues

* Added tests for EventsExpressExceptionFilterAttribute

* Added type asserts

* Fixed formatting

* Fix (#921)

* Issue page works correctly on first load and No result text locates correctly.

* Admin panel is not locked at middle of list.

* Button size fix.

* formating.

Co-authored-by: Hryn Oleksandr <Apostall107@users.noreply.github.com>

* Update TestCase.md

* Update UserStory.md

* Update UserStory.md

* try --legacy-peer-deps

* remove twitter login from clientapp

Co-authored-by: Hryn Oleksandr <37192098+Apostall107@users.noreply.github.com>
Co-authored-by: Hryn Oleksandr <Apostall107@users.noreply.github.com>
Co-authored-by: Vasyl Semeniuk <70913988+idontwannawakeup@users.noreply.github.com>
Co-authored-by: Halyna <39273210+mehalyna@users.noreply.github.com>
Co-authored-by: Oleksandr Savchuk <savchuk.sand0@gmail.com>

D1mon007 committed:
Delete Twitter Login from controllers and Tests

D1mon007 committed:
Merge development

D1mon007 committed:
Made method "ImageToByteArray" static
